run.sh

```diff
@@ -14,5 +14,5 @@ migrate)
     bundle exec rake db:schema:load
     ;;
 esac
-bundle exec import:all assets:precompile
+RAILS_ENV=production bundle exec import:all assets:precompile
```

Comment:
I wasn't telling you what to do! I was recounting what I did, to reproduce a production environment as closely as I could.

If this line does fix asset generation, then it might be that the environment is not set properly. `RAILS_ENV` **must** be set to `production` on production (and pre-production) servers.